plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/machine/HostedDockerInstanceProvider.java

```diff
@@ -121,6 +129,49 @@ protected void pullImage(MachineConfig machineConfig, String machineImageName, P
         MachineConfigImpl machineConfigForBuild = new MachineConfigImpl(machineConfig);
         machineConfigForBuild.setSource(new MachineSourceImpl(DockerInstanceProvider.DOCKER_FILE_TYPE)
                                                 .setContent("FROM " + machineConfig.getSource().getLocation()));
-        super.buildImage(machineConfigForBuild, machineImageName, true, progressMonitor);
+        buildImage(machineConfigForBuild, machineImageName, true, progressMonitor);
+    }
+
+    @Override
+    protected void buildImage(MachineConfig machineConfig,
```

Comment:
Add javadoc with description what special this method does. Don't forget to inherit docs